css-box-model/styles.css

```diff
@@ -0,0 +1,14 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+button {
+  margin-right: 12px;
+  margin-left: 16px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-radius: 6px;
+}
```

Comment:
Each button should have left and right padding:

---

![image](https://user-images.githubusercontent.com/16143771/176502204-1d5dd330-0294-4576-9353-481af40df92d.png)